docs/readme/readme_id.md

```
@@ -0,0 +1,226 @@
+Kami berusaha mencapai keseimbangan yang harmonis antara [prinsip sumber terbuka dan keberlanjutan](https://github.com/usebruno/bruno/discussions/269)
+
+Anda dapat menjelajahi [versi berbayar kami](https://www.usebruno.com/pricing) untuk melihat apakah ada fitur tambahan yang mungkin berguna bagi Anda atau tim Anda! <br/>
+
+## Daftar isi
+
+- [Instalasi](#instalasi)
+- [Bruno CLI](#bruno-cli)
+- [Jalankan dengan Docker](#jalankan-dengan-docker)
+- [Fitur](#fitur)
+  - [Berjalan di berbagai platform 🖥️](#berjalan-di-berbagai-platform-%EF%B8%8F)
+  - [Berkolaborasi melalui Git 👩‍💻🧑‍💻](#berkolaborasi-melalui-git-%E2%80%8D%E2%80%8D)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)